### DISCUSSION TITLE
Zigbee AND Bluetooth

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#16862] [Q&A]
Hello,

do i have to compile by myself to enable Bluetooth and Zigbee parallel?
Or does one of the precompiled versions both activations?

thx a lot!

**Comment by sfromis** [comment]
No such precompiled binaries. Not sure if that combo has been tested. 3 different types of radio in one box may not strengthen reliability.

**Comment by ChrisVonFrosta** [comment]
thx
